charts/gateway/values.yaml

```diff
@@ -9,6 +9,13 @@ license:
   accept: false
   # existingSecretName: ssg-license
 
+# Use set diskless Configuration mode for Gateway
+# when true , get values from environment variable and on false will get values from node.properties
+disklessConfig:
+  enabled: true
+  value:
```

Comment:
I wouldn't use the term "value" to store node.properties... it works for license because "license.value" refers to license.xml, but "disklessConfig.value" referring to node.properties looks awkward